app/controllers/crudController.js

```diff
@@ -0,0 +1,25 @@
+const errorService = require('../services/error');
+
+module.exports = name => {
```

Comment:
I think a better idea is to add a `constructor` to the `Controller` class and pass the service name in the constructor. Then you can export the class here and you instantiate it in the actual controller file.